metaflow/plugins/kfp/tests/flows/failure_flow.py

```diff
@@ -17,6 +23,12 @@ def start(self):
             print("let's succeed")
         self.next(self.no_retry)
 
+    @resources(
+        cpu="0.1",
+        cpu_limit="0.5",
+        memory="10M",
+        memory_limit="500M"
```

Comment:
Why are you setting the resources to this setting?  

This needs to be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) 

For example, if you want, you can have a `LimitRange` for the test namespace.  
```yaml
apiVersion: v1
kind: LimitRange
metadata:
  name: default-limitrange
spec:
  limits:
  - defaultRequest: # default request
      cpu: 100m
      memory: 128Mi
      ephemeral-storage: 100Mi
    default: # default limit
      cpu: 500m
      memory: 1G
      ephemeral-storage: 8G
```